front_end/panels/ai_chat/ui/settings/providers/LiteLLMSettings.ts

```
@@ -0,0 +1,558 @@
+        // Get counts from centralized getModelOptions
+        const allLiteLLMModels = this.getModelOptions('litellm');
+        const actualModelCount = litellmModels.length;
+        const hasCustomModels = allLiteLLMModels.length > actualModelCount;
+
+        // Get current mini and nano models from storage
+        const miniModel = getStorageItem(MINI_MODEL_STORAGE_KEY, '');
+        const nanoModel = getStorageItem(NANO_MODEL_STORAGE_KEY, '');
+
+        // Refresh existing model selectors with new options if they exist
+        if (this.miniModelSelector) {
+          refreshModelSelectOptions(this.miniModelSelector as any, allLiteLLMModels, miniModel, i18nString(UIStrings.defaultMiniOption));
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)